masu/providers/database/accessors.py

```diff
@@ -14,11 +14,16 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
-"""Initializer for external module."""
+"""Standardized namespacing entry-point for DB accessors."""
 
-# Provider Types
-AMAZON_WEB_SERVICES = 'AWS'
-
-# Compression formats
-UNCOMPRESSED = ''
-GZIP_COMPRESSED = 'GZIP'
+# pylint: disable=unused-import
```

Comment:
If we have different accessors they should just be in an accessors module not imported into an accessors file like this. Then you import with the standard package and module import process. See https://docs.python.org/3/tutorial/modules.html#packages